content/en/cluster.md

```diff
@@ -2,26 +2,22 @@
 title: Cluster
 status: Completed
 category: Concept
-tags: ["infrastructure", "fundamental", ""]
+tags: ["infrastructure", "fundamental", "architecture"]
```

Comment:
nit: do we alphabetize tags, I'm not sure order matters but we should be consistent if we do.

Reply:
apologies for the commenting here: @nate-double-u i noticed this in other contents while previously raising a pr, and no the tags are not alphabetize but in most them with two tags they are in order Z-A.

Reply:
Good point. I think we never agreed on something and a lot of terms don't have any tag. We could agree on something and update accordingly.

Reply:
Let's open an issue to discuss. No change would be needed here (we can make an update after our conversation if we decide a change is needed)

Reply:
I will put it on the agenda for the next maintainers meeting  and resolve this conversation